subsys/bluetooth/host/classic/br.c

```diff
@@ -801,7 +801,11 @@ int bt_br_init(void)
 	struct bt_hci_cp_write_inquiry_mode *inq_cp;
 	struct bt_hci_write_local_name *name_cp;
 	struct bt_hci_cp_write_class_of_device *cod;
+	struct bt_hci_rp_read_default_link_policy_settings *rp;
+	struct bt_hci_cp_write_default_link_policy_settings *policy_cp;
```

Comment:
Move this into the if-branch where it's actually used (variables should always be declared in the smallest possible scope)